src/Symfony/Component/Validator/Mapping/Factory/LazyLoadingMetadataFactory.php

```diff
@@ -90,17 +90,17 @@ public function getMetadataFor($value)
             return $this->loadedClasses[$class];
         }
 
+        if (!class_exists($class) && !interface_exists($class)) {
```

Comment:
juste on change please: false should be passed as 2nd arg to interface_exists (no need to trigger autoloader twice)

Reply:
done